sdk/core/src/main/java/com/klaviyo/coresdk/KlaviyoConfig.kt

```diff
@@ -17,13 +17,13 @@ class KlaviyoMissingContextException : Exception("You must add your application
 /**
  * Stores all configuration related to the Klaviyo Android SDK.
  */
-object KlaviyoConfig {
-    private const val NETWORK_TIMEOUT_DEFAULT: Int = 500
-    private const val NETWORK_FLUSH_INTERVAL_DEFAULT: Int = 60000
-    private const val NETWORK_FLUSH_DEPTH_DEFAULT: Int = 20
-    private const val NETWORK_FLUSH_CHECK_INTERVAL: Int = 2000
-    private const val NETWORK_USE_ANALYTICS_BATCH_QUEUE: Boolean = true
-    private val SYSTEM_CLOCK: Clock = SystemClock
+internal object KlaviyoConfig {
```

Comment:
what is the default access modifier in Kotlin? In Swift it's `internal` by default.

Reply:
seems to be public

Reply:
I think that is package private potentially. Assuming the test app is under the same package.

Reply:
test app is its own repo altogether, changing this to internal took access away from the test app, which was why I put the arguments in `Klaviyo.configure`